resolwe_bio/management/commands/generate_etc.py

```diff
@@ -0,0 +1,142 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import csv
+import gzip
+import json
+import os
+import random
+import datetime
+
+from django.core.management.base import BaseCommand
+from django.contrib.auth import get_user_model
+from django.conf import settings
+from django.utils import timezone
+from resolwe.flow.models import Data, DescriptorSchema, Process, Storage
+from resolwe_bio.tools.utils import gzopen
+
+
+class Command(BaseCommand):
+
+    """Generate ETC objects"""
+
+    help = "Generate ETC objects"
+
+    def add_arguments(self, parser):
+        parser.add_argument('-e', '--n-etc', type=int, default=3,
+                            help="Number of ETC objects to generate (default: %(default)s)")
+        parser.add_argument('--rseed', action='store_true', help="Use fixed random seed")
+
+
+    def get_superuser(self):
+        users = get_user_model().objects.filter(is_superuser=True).order_by('date_joined')
+        if not users.exists():
+            self.stderr.write("Admin does not exist: create a superuser")
+            exit(1)
+        return users.first()
+
+    def get_descriptor_schema(self, slug):
+        try:
+            ds = DescriptorSchema.objects.get(slug=slug)
+        except DescriptorSchema.DoesNotExist:
+            self.stderr.write("Descriptor schema ({}) does not exist: "
+                "try running './manage.py register' command".format(slug))
+            exit(1)
+        return ds
+
+
+    def get_process(self, slug):
+        try:
+            process = Process.objects.get(slug=slug)
```

Comment:
This will raise error if there are multiple `Processes` with same `slug`.

```
process_qs = Process.objects.filter(slug=slug)
if not process_qs.exists():
    self.stderr.write("Process ({}) does not exist: try running './manage.py register' command.".format(slug))
    exit(1)
return process_qs.order_by('version').last()
```